.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
+contact_links:
+  - name: Questions and Discussions
+    url: https://github.com/moby/buildkit/discussions/new
+    about: Use Github Discussions to ask questions and/or open discussion topics.
+  - name: Documentation
+    url: https://docs.docker.com/build/buildkit/
```

Comment:
I think we should point to the more-complete internal docs instead: https://github.com/moby/buildkit/tree/master/docs